ebuild-maintenance/git/text.xml

```diff
@@ -224,8 +221,8 @@ appropriately:
 
 <ul>
   <li>
-    <c>${CATEGORY}/${PN}:</c> Single Package (Note that <c>repoman commit</c>
-    and <c>pkgdev commit</c> will automatically insert this for you)
```

Comment:
Again, this is simply a fact, no need to remove this information.

Reply:
> We should distinguish between updating the recommended workflow, and documentation of the repoman tool. I don't see a compelling reason to remove all mentions of the latter.

I disagree. It's not a good idea to continue mentioning `repoman` in the devmanual when we want developers to stop using it.

Reply:
> Again, this is simply a fact, no need to remove this information.

But there's no value in keeping it. It only stands to lead readers to the false conclusion that repoman is a tool they should use.

Reply:
wfm